lib/commands/count.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var filter = require('../filter');
+var protocol = require('../protocol');
+
+module.exports = function doCount(socket, clientReqMsg) {
+  this.logger.info('doCount');
+  var dbName = clientReqMsg.fullCollectionName.replace('.$cmd', '');
+  var collection = this.mocks[dbName][clientReqMsg.query.count] || [];
```

Comment:
Good catch. I am not fully satisfied with the current API (want one to extract the collection name from the input object automatically, and an option to create collection for collection-changing commands such as `insert`) but this will have to do for now.